ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/OnChainPrivacyPrecompiledContract.java

```diff
@@ -165,7 +168,7 @@ public Bytes compute(final Bytes input, final MessageFrame messageFrame) {
           result);
     }
 
-    return result.getOutput();
+    return Bytes.EMPTY;
```

Comment:
Could this somehow introduce backwards incompatibility? I'm thinking of node that will resync in a network that already has processed PMTs... Could the public state end up being different because of this change?

It might be worth testing this!

What do you think @pinges ?

Reply:
I changed this and meant to revert it, though I'm interested in @pinges opinion.